factories/Event.php

```diff
@@ -23,15 +23,15 @@ public function __construct(Factory $faker)
     public function create($limit = 1, $flatten = false, $options = [])
     {
         for ($i = 1; $i <= $limit; $i++) {
-            $data[$i] = [
+            $data[$this->faker->dateTimeThisMonth('now')->format('Y-m-d')][$i] = [
                 'url' => $this->faker->url,
                 'title' => $this->faker->sentence(rand(6, 10)),
                 'date' => $this->faker->dateTimeThisMonth('now')->format('Y-m-d'),
                 'start_time' => $this->faker->dateTimeThisMonth('now')->format('H:i:s'),
                 'is_all_day' => $this->faker->boolean,
             ];
 
-            $data[$i] = array_replace_recursive($data[$i], $options);
+            //$data[$i] = array_replace_recursive($data[$i], $options);
```

Comment:
If this is no longer needed, let's remove the line completely.

Reply:
I wrote in my comment "In the factory, I had to comment out a line. My tests weren't passing but I'm not exactly sure what I need to do since I'm grouping by random date." 

It probably needs to be here, I'm just not sure how to handle it.

Reply:
I fixed it to allow options to be passed in and the tests pass with it uncommented.